### PR TITLE
[AIRFLOW-4681] Make sensors module pylint compatible

### PR DESCRIPTION
**Pull request by coopergillan**
Make sensor modules pylint compatible. Rename and use some local rather than class variables and add comments to ignore some pylint checks around higher impact code.

---
Issue link: [AIRFLOW-4681](https://issues.apache.org/jira/browse/AIRFLOW-4681)

Make sure to mark the boxes below before creating PR:

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

### CODE REVIEW

**Comment by coopergillan** [comment]
I did see https://github.com/apache/airflow/pull/5406, but noticed that it had been dormant for some time. I wanted to give it a shot from a fresh pull of `master`.

File: airflow/providers/jira/sensors/jira.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ class JiraTicketSensor(JiraSensor):
 
     :param jira_conn_id: reference to a pre-defined Jira Connection
     :type jira_conn_id: str
-    :param ticket_id: id of the ticket to be monitored
+    state:param ticket_id: id of the ticket to be monitored
```
Comment (mik-laj) [inline airflow/providers/jira/sensors/jira.py]:
```suggestion
    :param ticket_id: id of the ticket to be monitored
```
Travis is sad without this change.

Comment (coopergillan) [reply airflow/providers/jira/sensors/jira.py]: Nice catch, thank you! This snuck in somehow. Just pushed up a change.

**Comment by coopergillan** [comment]
Seeing [AIRFLOW-4681](https://issues.apache.org/jira/browse/AIRFLOW-4681) was marked as "Won't Do". Should we close this one?

**Comment by mik-laj** [comment]
No. Don't worry. This can be fixed later.

**Comment by coopergillan** [comment]
Build failures look unrelated and transient.

**Comment by potiuk** [comment]
Yep. I am working on the build errors in master still. I will let you know when to rebase when it is all fixed.

**Comment by potiuk** [comment]
Please rebase to latest master @coopergillan  - we had a failing master drama and there is much higher chance the build will succeed. We have still some intermittent errors, but there should be less often.

**Comment by coopergillan** [comment]
@potiuk @mik-laj - I just rebased from master, cleaned up some merge conflicts, and clarified some naming in the `HDFSSensor` module. All is squashed and pushed up. Thanks for the feedback!

**Comment by potiuk** [comment]
Some pylint problems still :(

**Comment by mik-laj** [comment]
It is unrelated changes.  I also saw it in another PR - https://github.com/apache/airflow/pull/7341

**Comment by kaxil** [comment]
> It is unrelated changes. I also saw it in another PR - #7341

Created https://github.com/apache/airflow/pull/7342 to fix it on master

**Comment by coopergillan** [comment]
Yep just pushed up a fixup for that one.

**Comment by coopergillan** [comment]
Will actually wait for #7342 and rebase on top of that one.

**Comment by coopergillan** [comment]
All right, this one should be good to go now. Let me know if it needs anything else. Thanks for the reviews.

**Review by potiuk — APPROVED** [review]
Cool. It seems to be good too go now !

**Comment by potiuk** [comment]
Thanks @coopergillan  !

**Pull request merged**